### PR TITLE
Addnewevent

### PR DESCRIPTION
**Pull request by JPCatze**
all function in addnewevent page is now complete (except extracting friend list from firestore, have to wait for other pages).

### CODE REVIEW

File: lib/src/utils/firestore/firestore.dart
Comment (INZDY) [inline lib/src/utils/firestore/firestore.dart:1]: Try catch, note that printing should not be put into production.

File: lib/src/features/addevent/addeventpeople.dart
Comment (INZDY) [inline lib/src/features/addevent/addeventpeople.dart:1]: Please note on your part that you need to come back and implement pulling people list from database.
If possible, separate sub sections or functions and put into other files, for readability.

File: lib/src/features/addevent/addevent.dart
[the remark this answers is not in the document]
Comment (INZDY) [reply lib/src/features/addevent/addevent.dart:1]: Try catch, note that printing should not be put into production.

**Review by INZDY — COMMENTED** [review]
Please refer to comments.

File: lib/src/utils/firestore/firestore.dart
Comment (JPCatze) [reply lib/src/utils/firestore/firestore.dart:1]: ok, I'll remove that part out

File: lib/src/features/addevent/addeventpeople.dart
Comment (JPCatze) [reply lib/src/features/addevent/addeventpeople.dart:1]: I already comment a small note, maybe I should add a note on top to make it clear enough. About separating sub section, I'll try to do that tomorrow.

**Pull request merged**